src/main/java/httpserver/HttpRequest.java

```diff
@@ -0,0 +1,29 @@
+package httpserver;
+
+public class HttpRequest{
+    private String route = "";
+
+    HttpRequest(HttpRequestBuilder builder){
+         this.route = builder.route;
+     }
+
+     public String getRoute(){
```

Comment:
In the domain of the HTTP spec, what is *route*?

Reply:
I think it might make sense if I phrase the question differently...

Does the HTTP spec mention routes? What word does it use?

Reply:
It looks like it's resource. I'll change it!